rtc/configuration.md

```diff
@@ -136,143 +136,198 @@ tinymce.init({
+#### Example of providing user details from your server
 
 ```js
 tinymce.init({
   selector: 'textarea', // change this value according to your HTML
   plugins: 'rtc',
-  rtc_initial_content_provider: ({documentId}) => {
-    return fetch(`/getContent/${documentId}`, {
+  rtc_user_details_provider: ({userId}) => {
+    return fetch('/getUserDetails', {
       method: 'POST',
       headers: {
         'Accept': 'application/json',
         'Content-Type': 'application/json'
-      }
+      },
+      body: JSON.stringify({userId})
     }).then(response => response.json());
   }
 })
 ```
 
-### `rtc_user_details_provider`
+## Optional configuration
 
-By default, a user's unique ID (the `sub` field from their [JWT](#rtc_token_provider)) will be displayed as their name in remote caret tooltips.
+### `rtc_snapshot`
 
-To display a descriptive name on the caret, the user ID needs to be resolved into user details that include the full user name. This resolution is done on each client to avoid sending any personal information through the RTC server.
+Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the regular content snapshots. These snapshots are not stored by the {{site.cloudname}} and must be handled by the integrator.
 
-Only the full name is required, but the whole object will be cloned and stored within the RTC caret information. It will then be included with both [`rtc_client_connected`](#rtc_client_connected) and [`rtc_client_disconnected`](#rtc_client_disconnected) callbacks so that the lookup does not need to be repeated.
+For any given document ID, the server guarantees the version number will only increase. It can be safely used for conflict resolution. For each document ID and version combination the snapshot content is guaranteed to be identical.
+
+The snapshot callback will be executed at regular intervals with access to the serialized editor content. The content is retrieved through a `getContent` function to reduce CPU load if the callback decides to not use the editor content.
 
-This provider function will be called once for each connecting client. Clients that reconnect may trigger a new call to the provider function rather than using cached data.
 
 **Type:** `Function`
 
 **Required:** no
 
-#### Input fields for `rtc_user_details_provider`
+#### Input fields for `rtc_snapshot`
 
 | Field | Type | Description |
 |-------|:----:|-------------|
-| `userId` | `string` | User ID to resolve into user details. |
+| `documentId` | `string` | The document ID from the [`rtc_document_id`](#rtc_document_id) option
+| `version` | `integer` | An increasing version number, specific to the current document ID, between 0 and 2147483648 (2<sup>31</sup>). |
+| `getContent()` | `string` | Function to execute to get the content for that particular version. |
```

Comment:
This looks a little weird, should it be

```suggestion
| `getContent` | `Function` | Function to execute to get the content for that particular version. |
```

Reply:
One of those things that snuck through a previous review and I've just moved it. That probably makes sense.